cmd/k8s-operator/deploy/chart/templates/deployment.yaml

```diff
@@ -35,9 +35,13 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
-      - name: oauth
-        secret:
-          secretName: operator-oauth
+        - name: oauth
+        {{- with .Values.oauthSecretVolume }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
I think the indent needs to be 10 here?

Reply:
Good spot... wonder why it works for me the way it is. Will check

Reply:
8 is actually correct. We want to be on the same level as `name`, not below

Reply:
I think the name is indented by 10 (the `-` sign is indented by 8)
It fails for me locally if I just uncomment the new `oauthSecretVolume` block and run `helm template` with
```
Error: YAML parse error on tailscale-operator/templates/deployment.yaml: error converting YAML to JSON: yaml: line 23: did not find expected '-' indicator
```
and succeeds if I change the indent to 10.

Reply:
Found the issue 😅 Had two versions of this, one in my IaC repo and one in the fork of your repo.
The one in my IaC repo looked like this:
```
      volumes:
      - name: oauth
```
Fixed it now, according to the style used throughout your `yaml`s